HW_2/task_4.py

```diff
@@ -0,0 +1,36 @@
+"""
+Задайте список из N элементов, заполненный целыми числами из промежутка [-N, N].
+Найдите произведение элементов на индексах, хранящихся в файле indexes.txt (в одной строке один индекс).
+Решение должно работать при любом натуральном N.
+
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+"""
+import random
+from random import randint
+
+number = int(input('Введите натуральное число N = '))
+my_list = [randint(-number, number) for _ in range(number)]
+print(my_list)
+
+res = 1
+with open('D:\Python_START-1\HW_2\indexes.txt', 'r') as file:
```

Comment:
Пожалуйста, используйте относительный путь.
```suggestion
with open('indexes.txt', 'r') as file:
```

Reply:
С относительным никак не получалось